.github/workflows/cache_data.yaml

```
@@ -33,13 +33,16 @@ jobs:
           # fetch all history so that setuptools-scm works
           fetch-depth: 0
 
+      # Install Micromamba with conda-forge dependencies
       - name: Setup Micromamba
-        uses: mamba-org/provision-with-micromamba@v15
+        uses: mamba-org/setup-micromamba@v1.2.1
```

A new version was released
```suggestion
        uses: mamba-org/setup-micromamba@v1.3.0
```

---

Done in 06c89e4367f09417949c49c0f4f9486931ad5af2